.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -201,15 +201,15 @@ def gen_docs_configs(xenial_parent_config):
         HiddenConf(
             "pytorch_cpp_doc_build",
             parent_build=xenial_parent_config,
-            filters=gen_filter_dict(branches_list=["master", "nightly"],
+            filters=gen_filter_dict(branches_list=["master", "main", "nightly"],
                                     tags_list=RC_PATTERN),
         )
     )
     configs.append(
         DocPushConf(
             "pytorch_cpp_doc_push",
             parent_build="pytorch_cpp_doc_build",
-            branch="master",
+            branch="main",
```

Comment:
@malfet I wasn't sure what to do here. From what I can tell, it would be tough to support both main and master for this. If I added both configs, I thought it might push the docs twice for each branch